### PR TITLE
Fix bug of update patient command

### PR DESCRIPTION
**Pull request by kkeejjuunn**
User can update patient using command "update patient #<id> <Name/NRIC/Room> <New info>"
image

### CODE REVIEW

File: src/main/java/duke/command/UpdatePatientCommand.java [hunk 1]
```diff
@@ -12,32 +12,41 @@
...
-                targetPatient.setName(updatedValue);
-            } else if (targetInfo.equals("nric")) {
-                targetPatient.setNric(updatedValue);
-            } else if (targetInfo.equals("room")) {
-                targetPatient.setRoom(updatedValue);
-            } else {
-                throw new DukeException("No such Patient information existed. Please Enter a valid Patient information");
+        String[] tempCommand = command.split(" ", 3);
+        char firstChar = tempCommand[0].charAt(0);
+        if (firstChar == '#'){
```
Comment (HUANGXUANKUN) [inline src/main/java/duke/command/UpdatePatientCommand.java]: There is no else statement for this condition. Which means, for command `update patient 1 Name Huang`, nothing will be done

Comment (HUANGXUANKUN) [reply src/main/java/duke/command/UpdatePatientCommand.java]: I suggest to check for existence of '#' in `Parser.parseUpdatePatient()` if the UpdatePatientCommand only supports update by id for now. If the '#' is missing, throw an exception to remind user instead.

Comment (HUANGXUANKUN) [reply src/main/java/duke/command/UpdatePatientCommand.java]: Or you can support both 
1. `update patient #<id> <header> <value>`
2. `update patient <name> <header> <value>` (using the same approach as in deleteTask/deletePatient)

Comment (kkeejjuunn) [reply src/main/java/duke/command/UpdatePatientCommand.java]: Okay. For now I will just throw exception first. Have made the changes.

**Pull request merged**